tools/modules/framework/interface_package.sh

```
@@ -0,0 +1,171 @@
+
+framework_options+=(
+	["pkg_installed,author"]="@dimitry-ishenko"
+	["pkg_installed,desc"]="Check if package is installed"
+	["pkg_installed,options"]="<apt package name>"
+	["pkg_installed,feature"]="pkg_installed"
+	["pkg_installed,group"]="Interface"
+)
+
+pkg_installed()
+{
+	local status=$(dpkg -s "$1" 2>/dev/null | sed -n "s/Status: //p")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)